setup.cfg

```
@@ -88,7 +88,7 @@ docs =
     PyYAML>=3.13
     scipy>=1.1
     matplotlib>=3.1,!=3.4.0
-    sphinx-changelog
+    sphinx-changelog>=1.1.0rc1
```

Do we still need to pin to RC? Can we pin to stable yet?

---

Well this is the minimum and it's technically correct, tho we can bump to 1.1.0.

```suggestion
    sphinx-changelog>=1.1.0
```